libfreerdp/core/window.c

```diff
@@ -350,6 +352,14 @@ BOOL update_read_window_state_order(wStream* s, WINDOW_ORDER_INFO* orderInfo, WI
 
 BOOL update_read_window_icon_order(wStream* s, WINDOW_ORDER_INFO* orderInfo, WINDOW_ICON_ORDER* window_icon)
 {
+	if (window_icon->iconInfo)
+	{
+		free(window_icon->iconInfo->bitsColor);
```

Comment:
same remark here, it would be neat to have a function to free the icon info and also use it above.